tests/trading/test_cancelOrder.py

```diff
@@ -109,19 +109,13 @@ def test_exceptions(contractsFixture):
     orderID = makeOrder.publicMakeOrder(orderType, fxpAmount, fxpPrice, market.address, outcomeID, longTo32Bytes(0), longTo32Bytes(0), tradeGroupID, sender=tester.k1)
     assert(orderID != bytearray(32)), "Order ID should be non-zero"
 
-    # Permissions exceptions
-    with raises(TransactionFailed):
-        cancelOrder.cancelOrder(tester.a1, orderID, orderType, market.address, outcomeID, sender=tester.k1)
-    with raises(TransactionFailed):
-        cancelOrder.refundOrder(tester.a1, orderType, 0, fxpAmount, market.address, outcomeID, sender=tester.k1)
```

Comment:
I removed these two tests because `cancelOrder()` used to be a function that was not public.  (Now `publicCancelOrder()` and `cancelOrder()` have been combined.)  Also, `refundOrder()` is private, and so this line was giving an error saying that cancelOrder did not contain `refundOrder()` (as opposed to a transaction failure when using Serpent).